src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,5 +11,9 @@ public class CliSyntax {
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
```

Comment:
We should rename the prefixes to distinguish between the ones for Employee and the ones for Leaves

Reply:
so just "email/ "? or do we only change the one for leaves, because for add command, all prefix seems to be a single character?

Reply:
as in the names of the constants - something like PREFIX_PERSON_EMAIL